pages/blueprint/index.js

```diff
@@ -248,9 +200,9 @@ class BlueprintPage extends React.Component {
                       </button>
                     </dd>}
                   <dt>Install size</dt>
-                  <dd>2,678 KB</dd>
+                  <dd></dd>
                   <dt>Last modified date</dt>
-                  <dd>Thu,  9 Nov 2017</dd>
+                  <dd></dd>
```

Comment:
Should we just remove this chunk of html altogether until we're actually pulling data in? At the moment, I don't think there's api support for showing install size either.
```
<dt>Install size</dt>
<dd></dd>
<dt>Last modified date</dt>
<dd></dd>
```